analysis/tests/src/CompletionPattern.res

```diff
@@ -105,3 +105,12 @@ ignore(b)
 
 // switch b { | #three({})}
 //                      ^com
+
+let c: array<bool> = []
```

Comment:
What's the idea about pattern matching for arrays? Are there useful patterns for it?

Reply:
It's mostly for completeness - if you get completions almost everywhere in patterns I think you'll expect to get it with arrays too, even if it's not as useful as the others.